.github/workflows/push-pull-main.yml

```diff
@@ -131,10 +105,9 @@ jobs:
         env-prefix: ${{ matrix.prefix }}
         env-label: ${{ matrix.label }}
         env-files: ${{ inputs.tests-env-files }}
-        test-dir: tests/unit
+        test-dir: tests
```

Comment:
As discussed in #32, I'm ok with variableizing this, but I absolutely do not want to _encourage_ the cluttered pattern of having multiple tests dirs in the home dir.